sotodlib/toast/workflows/sim_observe.py

```diff
@@ -97,6 +97,10 @@ def setup_simulate_observing(parser, operators):
             detset_key="pixel",
             session_split_key="wafer_slot",
             enabled=False,
+            # Only raise the second bit for turnarounds.
+            # Marking them as invalid breaks the
+            # atmospheric simulation
+            turnaround_mask=2,
```

Comment:
The current defaults in the `SimGround` operator [set the turnarounds to "unstable_scanrate"](https://github.com/hpc4cmb/toast/blob/697eefd199d61c25c082535078875bd3973256bf/src/toast/ops/sim_ground.py#L301) (equivalent to `4`, not invalid).  The defaults of the `SimAtmosphere` operator [is to use the invalid mask](https://github.com/hpc4cmb/toast/blob/697eefd199d61c25c082535078875bd3973256bf/src/toast/ops/sim_tod_atm.py#L81) (equivalent to `1`).  So I don't see any conflict in the default flagging...

Is it possible there is a stale config somewhere with hardcoded flag values that is interfering with the defaults?